feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts

```diff
@@ -184,6 +226,15 @@ export class ConfiguratorTabBarComponent {
     });
     return tabIndex;
   }
+  /**
+   * Returns the tabindex for the overview tab.
+   * The overview tab is excluded from the tab chain if currently the configuration page is displayed.
+   * @param pageType
```

Comment:
The same here:
Please add an empty line between an explanation and a  return value.